renovate.json

```diff
@@ -5,18 +5,26 @@
   "lockFileMaintenance": {
     "enabled": true
   },
+  "semanticCommitType": "chore",
+  "semanticCommitScope": "dependencies",
   "packageRules": [
     {
       "packagePatterns": ["^@untool/"],
       "packageNames": ["untool"],
       "groupName": "Update untool monorepo packages"
     },
     {
+      "depTypeList": ["dependencies", "peerDependencies"],
       "semanticCommitType": "fix",
-      "semanticCommitScope": "dependencies",
-      "depTypeList": ["dependencies", "peerDependencies"]
+      "semanticCommitScope": "dependencies"
```

Comment:
Can we not get rid of the scope altogether?

Reply:
Yes, we could. I'll remove it.